test_runner/src/test/kotlin/ftl/fixtures/test_app_cases/flank-single-gameloop-ios.yml

```diff
@@ -0,0 +1,10 @@
+gcloud:
+  app: test_projects/ios/FlankGameLoopExample/ipa/FlankGameLoopExample.ipa
+  type: game-loop
+  scenario-numbers:
+      - 1
+  device:
+      - model: iphone11
+        version: 13.6
+        orientation: portrait
+        locale: en_US
```

Comment:
- When I try to run this test from Debug.kt Flank returns

```
Both of following options must be specified [test, xctestrun-file].

Process finished with exit code 2

```

- Blank line missing

Reply:
@adamfilipow92 FTL with gameloop mode doesn't need xctestrun-file,
however Flank currently requires it. @Sloox will fix this in next a PR

Reply:
Thanks! I try to run in on gcloud cli!

Reply:
@adamfilipow92 this wont work until @jan-gogo applies a fix that will come in #1341 . However #1341 is blocked until other tickets are closed (including this one.)